plugins/woocommerce-admin/client/layout/controller.js

```diff
@@ -173,6 +176,25 @@ export const getPages = () => {
 		} );
 	}
 
+	if ( isFeatureEnabled( 'marketplace' ) ) {
+		pages.push( {
+			container: Marketplace,
+			layout: {
+				header: false,
```

Comment:
**Non-review question:** 

Is this a simple way to disable the WC Admin top bar on this page? If so, this is handy!

Reply:
Yes it is! The CSS still leaves space for a header at the top of the page, but we can fill that with our own header.